src/js/me-i18n-locale-ar.js

```diff
@@ -0,0 +1,74 @@
+/*!
+ * This is a i18n.locale language object.
+ *
+ * Arabic
+ *
+ * @author
+ *   rotti3
+ *   Sascha Greuel (Twitter: @SoftCreatR)
+ *
+ * @see
+ *   me-i18n.js
+ *
+ * @params
+ *  - exports - CommonJS, window ..
+ */
+(function (exports) {
+    "use strict";
+
+    if (exports.ar === undefined) {
+        exports.ar = {
+            "mejs.plural-form": 1,
+
+            // me-shim
+            "mejs.download-file": "تنزيل الملفات",
+
+            // mep-feature-contextmenu
+            "mejs.fullscreen-off": "غلق الشاشة الكاملة",
+            "mejs.fullscreen-on": "عرض الشاشة الكاملة",
+            "mejs.download-video": "تنويل الفيديو",
+
+            // mep-feature-fullscreen
+            "mejs.fullscreen": "شاشة كاملة",
+
+            // mep-feature-jumpforward
+            "mejs.time-jump-forward": ["قفز ثانية للأمام", "قفز للأمام %1 ثواني"],
+
+            // mep-feature-playpause
+            "mejs.play": "تشغيل",
+            "mejs.pause": "توقيف",
+
+            // mep-feature-postroll
+            "mejs.close": "غلق",
+
+            // mep-feature-progress
+            "mejs.time-slider": "التحكم بتوقيت العرض",
+            "mejs.time-help-text": "إستخدم زر الأيمين والأيسر للقفز ثانية, إستخدم زر الأعلى والأسفل للقفز 10 ثواني.",
+
+            // mep-feature-skipback
+            "mejs.time-skip-back": ["الرجوع للخلف 1 ثانية", "الرجوع للخلف %1 ثواني"],
```

Comment:
It should be %1; and as far as I know, in Arabic there are 6 different possibilities for pluralization and I see only 2, can you add the rest, please?